app/views/instruments/_instrument_fields.html.haml

```diff
@@ -70,3 +70,6 @@
   = f.input :lock_window,
     label: text("instruments.instrument_fields.reservation.label.lock_window"),
     hint: text("instruments.instrument_fields.reservation.instruct.lock_window")
+  = f.input :cutoff_hours,
+    label: text("instruments.instrument_fields.reservation.label.cutoff_hours"),
```

Comment:
Would it be worth moving some of these locales onto the model so simple_form can use `human_attribute_name`, which is one of the things it tries to use for `label`.